tests/test_main.py

```diff
@@ -343,3 +344,17 @@ def test_format_linter_report(errors_linter, errors_mate):
         f"Function 'format_linter_report' should return {errors_mate} "
         f"when 'errors' equals to {errors_linter}"
     )
+
+
+def test_comment_deleted():
+    lines = inspect.getsource(main)
+    assert "# write your code here" not in lines, (
+        "Remove the unnecessary" " comment '# write your code here'"
+    )
+
+
+def test_double_quotes_instead_of_single():
+    lines = inspect.getsource(main)
+    assert "'" not in lines, (
+        'You have to use a double quotes "" instead' " of single ''"
+    )
```

Comment:
The test `test_double_quotes_instead_of_single` is enforcing the use of double quotes over single quotes. Make sure this requirement is specified in the task description or checklist, as it might be a stylistic preference rather than a functional requirement.